proto/dymension/eibc/query.proto

```diff
@@ -14,6 +15,10 @@ service Query {
   rpc Params(QueryParamsRequest) returns (QueryParamsResponse) {
     option (google.api.http).get = "/dymensionxyz/dymension/eibc/params";
   }
+  // Queries a Demand Order by id.
```

Comment:
if u already adding queries, what about `AllPendingOrders`

Reply:
just did it for debug purposes for the test but can add it on a different PR.